README.md

```diff
@@ -34,6 +34,14 @@ type.
 **Define styles in a stylesheet**
 
 ```swift
+enum Style: String, StringConvertible {
```

Comment:
@onmyway133 Swift also has this protocol:

```swift
public protocol CustomStringConvertible {
    /// A textual representation of `self`.
    public var description: String { get }
}
```

So I think `Convertible` suffix is already confusing 😄